cfme/dashboard.py

```diff
@@ -112,36 +137,66 @@ def time_next(self):
     @property
     def is_minimized(self):
         self.close_zoom()
-        return not sel.is_displayed(self._minimize.format(self._div_id))
+        if not self.newer_version:
+            return not sel.is_displayed(self._minimize.format(self._div_id))
+        else:
+            return not sel.is_displayed(self._content.format(self._div_id))
 
     @property
     def can_zoom(self):
         """Can this Widget be zoomed?"""
-        return sel.is_displayed(self._zoom.format(self._div_id))
+        self.close_zoom()
+        if not self.newer_version:
+            return sel.is_displayed(self._zoom.format(self._div_id))
+        else:
+            self.open_dropdown_menu()
+            zoomable = sel.is_displayed(self._menu_zoom.format(self._div_id))
+            self.close_dropdown_menu()
+            return zoomable
 
     def remove(self, cancel=False):
         """Remove this Widget."""
         self.close_zoom()
-        sel.click(self._remove.format(self._div_id), wait_ajax=False)  # alert
-        sel.handle_alert(cancel)
+        if not self.newer_version:
+            sel.click(self._remove.format(self._div_id), wait_ajax=False)  # alert
+            sel.handle_alert(cancel)
+        else:
+            self.open_dropdown_menu()
+            sel.click(self._menu_remove.format(self._div_id))
+            self.close_dropdown_menu()
 
     def minimize(self):
         """Minimize this Widget."""
         self.close_zoom()
         if not self.is_minimized:
-            sel.click(self._minimize.format(self._div_id))
```

Comment:
I was looking at the if/else and wondering if it made sense to functionalize that and pass in the click function, but..........